CI fails: AttributeError: 'str' object has no attribute '__module__'

CI fails:
> AttributeError: 'str' object has no attribute '__module__'

> FAILED tests/test_models.py::test_flatten_messages_as_text_for_all_models[MLXModel-model_kwargs6-patching6-True] - AttributeError: 'str' object has no attribute '__module__'. Did you mean: '__mod__'?

Traceback:
```python
>               stack.enter_context(patch(target, **kwargs))

tests/test_models.py:853: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/opt/hostedtoolcache/Python/3.12.13/x64/lib/python3.12/contextlib.py:526: in enter_context
    result = _enter(cm)
             ^^^^^^^^^^
/opt/hostedtoolcache/Python/3.12.13/x64/lib/python3.12/unittest/mock.py:1451: in __enter__
    self.target = self.getter()
                  ^^^^^^^^^^^^^
/opt/hostedtoolcache/Python/3.12.13/x64/lib/python3.12/pkgutil.py:513: in resolve_name
    mod = importlib.import_module(modname)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/opt/hostedtoolcache/Python/3.12.13/x64/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
<frozen importlib._bootstrap>:1387: in _gcd_import
    ???
<frozen importlib._bootstrap>:1360: in _find_and_load
    ???
<frozen importlib._bootstrap>:1331: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:935: in _load_unlocked
    ???
<frozen importlib._bootstrap_external>:999: in exec_module
    ???
<frozen importlib._bootstrap>:488: in _call_with_frames_removed
    ???
/opt/hostedtoolcache/Python/3.12.13/x64/lib/python3.12/site-packages/mlx_lm/__init__.py:9: in <module>
    from .convert import convert
/opt/hostedtoolcache/Python/3.12.13/x64/lib/python3.12/site-packages/mlx_lm/convert.py:11: in <module>
    from .utils import (
/opt/hostedtoolcache/Python/3.12.13/x64/lib/python3.12/site-packages/mlx_lm/utils.py:41: in <module>
    from .tokenizer_utils import TokenizerWrapper
/opt/hostedtoolcache/Python/3.12.13/x64/lib/python3.12/site-packages/mlx_lm/tokenizer_utils.py:505: in <module>
    AutoTokenizer.register("NewlineTokenizer", fast_tokenizer_class=NewlineTokenizer)
/opt/hostedtoolcache/Python/3.12.13/x64/lib/python3.12/site-packages/transformers/models/auto/tokenization_auto.py:984: in register
    TOKENIZER_MAPPING.register(config_class, tokenizer_class, exist_ok=exist_ok)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = _LazyAutoMapping(), key = 'NewlineTokenizer'
value = <class 'mlx_lm.tokenizer_utils.NewlineTokenizer'>, exist_ok = False

    def register(self, key: type[PreTrainedConfig], value: _LazyAutoMappingValue, exist_ok=False) -> None:
        """
        Register a new model in this mapping.
        """
        if hasattr(key, "__name__") and key.__name__ in self._reverse_config_mapping:
            model_type = self._reverse_config_mapping[key.__name__]
            if model_type in self._model_mapping and not exist_ok:
                raise ValueError(f"'{key}' is already used by a Transformers model.")
    
        # Some remote code may simply register a new custom model/processor/..., while using a native Transformers config. In such
        # cases, we should skip registering, as we will otherwise always remap the native config to the custom model/processor/... in
        # the same session, even if `trust_remote_code=False` is specified by the user (in which case we should use the native
        # Transformers model/processor/... corresponding to the config)
        # This is because remote/native is indistinguisable from the config class only in such cases, as they both use the same class - then
        # `from_pretrained`/`from_config` are responsible to grab the correct class depending on whether `trust_remote_code` is True/False
>       if key.__module__.startswith("transformers."):
           ^^^^^^^^^^^^^^
E       AttributeError: 'str' object has no attribute '__module__'. Did you mean: '__mod__'?

/opt/hostedtoolcache/Python/3.12.13/x64/lib/python3.12/site-packages/transformers/models/auto/auto_factory.py:680: AttributeError
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -78,7 +78,7 @@ toolkit = [
 ]
 transformers = [
   "accelerate",
-  "transformers>=4.0.0",
+  "transformers>=4.0.0,!=5.13.0",  # !=5.13.0 (see GH-2487)
   "smolagents[torch]",
 ]
 vision = [

